e2e-win/pwsh_command_not_found.Tests.ps1

```
@@ -0,0 +1,104 @@
+                        if ($Name -ne 'mise-probe-missing-tool') { return }
+                        if ($global:__mise_probe_guarded) {
+                            if (Test-Path -Path Function:\_mise_probe_hook_absent) { _mise_probe_hook_absent }
+                        } else {
+                            _mise_probe_hook_absent
+                        }
+                        $global:__mise_probe_reached = $true
+                        $eventArgs.Command = Get-Command Get-Date
+                        $eventArgs.StopSearch = $true
+                    }
+                }
+                try { mise-probe-missing-tool | Out-Null } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)